samples/sampler/stories/qa/Scroller.js

```diff
@@ -1284,3 +1284,35 @@ export const WithContentContainerOnFocus = () => {
 };
 
 WithContentContainerOnFocus.storyName = 'With Content Container';
+
+export const WithFixedPopupPanels = () => {
+	const [open, setOpenState] = useState(false);
+
+	const handleOpen = useCallback(() => {
+		setOpenState(true);
+	}, [setOpenState]);
```

Comment:
We don't have to add setter as dependency here.
Please refer to https://react.dev/reference/react/useState#setstate-caveats.